src/main/java/com/conveyal/datatools/manager/jobs/MergeStrategy.java

```diff
@@ -0,0 +1,46 @@
+package com.conveyal.datatools.manager.jobs;
+
+/**
+ * This enum defines the different strategies for merging, which is currently dependent on whether trip_ids and/or
+ * service_ids between the two feeds are exactly matching.
+ */
+public enum MergeStrategy {
+    /**
+     * If service_ids and trip_ids between active and future feed are all unique, all IDs shall be included
+     * in merged feed. If a service_id from the active calendar has end_date in the future, the end_date shall be
+     * set to one day prior to the earliest start_date in future dataset before appending the calendar record to
+     * the merged file. It shall be ensured that trip_ids between active and future datasets must not match.
+     */
+    DEFAULT,
+    /**
+     * If service_ids and trip_ids in active feed are the same as future feed then the service end date for the
+     * merged feed shall match with future feed’s service end date and the service start date for the merged feed
+     * should be the merged date. All files from the future feed only shall be used in the merged feed.
+     */
+    EXTEND_FUTURE,
+    /**
+     * If trip_ids provided in active and future feeds are the same but the service_ids are unique then merge
+     * functionality shall reject feeds from merging. The user shall be notified that a new service requires unique
+     * trip_ids for merging.
+     */
+    FAIL_DUE_TO_MATCHING_TRIP_IDS,
```

Comment:
This one seems really odd to me since it doesn't influence any merging behavior and is just a stopgap so that merging doesn't even start in the first place. I think it'd be better to just set the mergeFeedResult to have failed and not use this as a "Strategy".

Reply:
This is a good one. I removed this enum entry in https://github.com/ibi-group/datatools-server/pull/376/commits/b87b5756f90c6a2c721f65018a7e3c8cc4ef67cb, with some other refactors.